soda/src/main/java/it/okkam/opendata/models/soda/SodaResource.java

```diff
@@ -182,5 +186,37 @@ public String getContact_email() {
   public void setContact_email(String contact_email) {
     this.contact_email = contact_email;
   }
-  
+
+  public String getLens_view_type() {
+    return lens_view_type;
+  }
+
+  public void setLens_view_type(String lens_view_type) {
+    this.lens_view_type = lens_view_type;
+  }
+
+  public String getBlob_mime_type() {
+    return blob_mime_type;
+  }
+
+  public void setBlob_mime_type(String blob_mime_type) {
+    this.blob_mime_type = blob_mime_type;
+  }
+
+  public boolean isHide_from_data_json() {
+    return hide_from_data_json;
+  }
+
+  public void setHide_from_data_json(boolean hide_from_data_json) {
+    this.hide_from_data_json = hide_from_data_json;
+  }
+
+  public String getPublication_date() {
+    return publication_date;
+  }
+
+  public void setPublication_date(String publication_date) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this local variable to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://dev.okkam.it/sonar/coding_rules#rule_key=squid%3AS00117)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://dev.okkam.it/sonar/coding_rules#rule_key=squid%3AS00100)